### ISSUE TITLE
Blocking Wanda Fireball.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Yahkal/replicaisland#39]
```
What steps will reproduce the problem?
1. In level_2_6_grass after talking Wanda the first time, she moves right to 
unblock the way.
2. Run right to pass her before she shots.

What is the expected output? What do you see instead?
She never fires the ball, or the fireball never hits the wall. You get stuck 
there (need to restart 
the game)

What version of the product are you using? On what operating system?
Not sure about it, I think the last version. Android 1.6 in HTC Magic

Please provide any additional information below.
Not sure about the level name, but at least in what happens just after the 
dialog Wanda_2_6_1_1.

```

Original issue reported on code.google.com by `fran....@gmail.com` on 14 Apr 2010 at 4:17
- Merged into: #5
